index.html

```diff
@@ -1,8 +1,165 @@
+                                  id="path38" fill="#cabd4b"></path>
+                            <path class="st5"
+                                  d="M163 196l50 51c-22 14-42 33-56 56l-51-51c16-22 35-41 57-56z"
+                                  id="path40" fill="#cabd4b"></path>
+                            <path class="st6"
+                                  d="M106 252l51 51c-10 16-18 34-22 53H63c6-37 21-73 43-104z"
+                                  id="path42" fill="#66a2f6"></path>
+                            <path class="st6"
+                                  d="M16 356h47c-4 26-4 53 0 80H16l-4-5-3-5-3-5-2-5-2-5-1-5-1-5v-10l1-5 1-5 2-5 3-5 2-5 4-5z"
+                                  id="path44" fill="#66a2f6"></path>
+                            <path class="st6"
+                                  d="M506 540l-51-51c10-16 18-34 22-53h72c-6 37-21 73-43 104z"
+                                  id="path46" fill="#66a2f6"></path>
+                            <path class="st6"
+                                  d="M612 391v10l-1 5-2 5-1 5-3 5-2 5-4 5-3 5h-47c3-26 3-52 0-79h47l4 4 3 5 3 5 2 5 2 5 1 5z"
+                                  id="path48" fill="#66a2f6"></path>
+                            <path class="st7"
+                                  d="M140 433l-5 3c-6-27-8-55 0-80l5 2 4 3 5 2 4 3 4 2 3 3 3 2 3 3 3 2 2 3 2 2 2 3 1 2 1 3 1 2v3l-1 2v3l-1 2-1 3-2 2-2 3-2 2-3 3-3 2-3 3-3 2-4 3-4 2-5 3zm20-37c0-7-5-12-12-12s-13 5-13 12 6 13 13 13 12-6 12-13z"
+                                  id="path50" fill="#247eff"></path>
+                            <path class="st7"
+                                  d="M63 356h72c-5 26-4 52 0 80H63c-4-27-4-54 0-80z"
+                                  id="path52" fill="#247eff"></path>
+                            <path class="st7"
+                                  d="M477 357c12 26 12 53 0 79l-5-2-4-3-5-2-4-3-4-2-3-3-3-2-3-3-3-2-2-3-2-2-2-3-1-2-1-3v-2l-1-3 1-2v-3l1-2 1-3 2-2 2-3 2-2 3-3 3-2 3-3 3-2 4-3 4-2 5-3 4-2zm0 39a13 13 0 1 0-25 0 13 13 0 0 0 25 0z"
+                                  id="path54" fill="#247eff"></path>
+                            <path class="st7"
+                                  d="M549 357c4 26 4 53 0 79h-72c6-26 6-53 0-79z"
+                                  id="path56" fill="#247eff"></path>
+                            <path class="st2"
+                                  d="M346 106v47c-27-4-52-4-80 0v-47l2-2 2-1 1-2 3-1 2-2 2-1 2-1 3-2h2l3-1 3-1 3-1h6l6-1 6 1h7l2 1 3 1 3 1 3 1 2 1 2 1 3 1 2 2 2 1 2 2 2 2z"
+                                  id="path58" fill="#7f7fe4"></path>
+                        </g>
+                    </g>
+                </g>
+            </svg>
+            <h1>RustFest Netherlands - Q2 2020</h1>
+            <p>
+                The RustFest Netherlands committee is working hard behind the
+                scenes on getting everything ready. We hope to tell you more
+                soon so keep an eye on this page!
+            </p>
```

Comment:
What do you think about a link to https://twitter.com/rustfest and the [blog](https://blog.rustfest.eu/)?